api/src/prescription/campaign-participation/domain/usecases/index.js

```diff
@@ -1,23 +1,41 @@
 import { dirname, join } from 'node:path';
 import { fileURLToPath } from 'node:url';
 
+import * as badgeAcquisitionRepository from '../../../../../lib/infrastructure/repositories/badge-acquisition-repository.js';
```

Comment:
on pourrait mettre un todo use API for this import . on est pas censé attauqer les badge directement . ça nous permettra de voir ce qu'il nous manque comme API.

Ou formaliser : `// TODO : NEED INTERNAL API` pour tout les repo qui nécessiterait d'être remplacé par une api interne ?

Reply:
On en à parlé avec @frinyvonnick il me semble. Quand la migration sera terminée, on pourra grace à ces imports effectivement voir quelles API internes on à besoin. 

Si le commentaire va à tout le monde, ça me va de le rajouter !